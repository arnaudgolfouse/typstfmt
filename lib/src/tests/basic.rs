use super::make_test;

make_test!(unchanged, " ");
make_test!(one_space, "  ");
make_test!(one_space_b, "   ");
make_test!(two_line_max, "\n\n\n");
make_test!(let_stmt_unchanged, "#let ident = variable");
make_test!(let_stmt_period_terminated, "#let ident = variable;");
make_test!(let_stmt_no_spacing, "#let ident=variable");
make_test!(
    content_block_spaced,
    "#[\n_Glaciers_ form an important part \nof the earth's climate system.]"
);
make_test!(ten_adds, &format!("#{{{}1}}", "1+".repeat(10)));
make_test!(thirty_adds, &format!("#{{{}1}}", "1+".repeat(30)));
make_test!(markup_block_spaced, "[ 4 ]");
make_test!(markup_block_too_spaced, "[  4  ]");
make_test!(markup_block_tight, "[4]");
make_test!(markup_block_only_space, "[   ]");
make_test!(just, "#let page_turned = page not in header_pages");
