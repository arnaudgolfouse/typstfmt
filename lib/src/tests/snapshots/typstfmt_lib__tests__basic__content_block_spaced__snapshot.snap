---
source: lib/src/tests/basic.rs
description: "INPUT\n===\n\"#[\\n_Glaciers_ form an important part \\nof the earth's climate system.]\"\n===\n#[\n_Glaciers_ form an important part \nof the earth's climate system.]\n===\nFORMATTED\n===\n#[\n_Glaciers_ form an important part \nof the earth's climate system.]"
expression: formatted
---
"#[\n_Glaciers_ form an important part \nof the earth's climate system.]"
